### PR TITLE
Lazik234/fix change team

### PR DESCRIPTION
**Pull request by lazik234**

### CODE REVIEW

File: FixCode/FixCode.m [hunk 1]
```diff
@@ -127,12 +127,17 @@ - (void)swizzleCodeSigningResolution
     if (error) {
         NSLog(@"Error: %@", error);
     }
-
-    [objc_getClass("Xcode3CodesignTroubleshootingViewController") aspect_hookSelector:@selector(viewDidAppear) withOptions:AspectPositionAfter usingBlock:^(id<AspectInfo> info) {
```
Comment (KrauseFx) [inline FixCode/FixCode.m:131]: Don't we need both this and the new code?

Comment (neonichu) [inline FixCode/FixCode.m:131]: I think the idea is that the old code would no longer be needed.

**Comment by neonichu** [comment]
Thanks for the contribution! :tada:

**Comment by KrauseFx** [comment]
Awesome, do we get a new release? :rocket:

**Comment by neonichu** [comment]
:shipit: https://github.com/neonichu/FixCode/releases/tag/0.3.0

**Pull request merged**